### PR TITLE
Prepare new release

### LINKED ISSUE #97: Release v2019.8.4

### PR DESCRIPTION
**Pull request by andersy005**
- Add fsspec to list of dependencies
- Closes #97 

~~Waiting on https://github.com/intake/intake-xarray/pull/46 to be merged ...~~

### CODE REVIEW

File: docs/environment.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: intake-esm-dev
 channels:
   - conda-forge
 dependencies:
-  - dask
+  - dask>=2.2
```
Comment (martindurant) [inline docs/environment.yml]: This should already bring in fsspec. Let us know if this is not the case.

Comment (andersy005) [reply docs/environment.yml]: You are right.. I just fixed it! Thank you

**Comment by martindurant** [comment]
done https://github.com/conda-forge/intake-xarray-feedstock/pull/3

**Comment by andersy005** [comment]
Thank you @martindurant!

**Pull request merged**